app/gameengine.py

```diff
@@ -5623,15 +5741,34 @@ def handle_player_resign(self, player_id):
 
         return True
 
-    def holomem_can_be_attached_with_sub_type(self, holomem, sub_type: str) -> bool:
-        return all([card.get("sub_type") != sub_type for card in holomem["attached_support"]])
+    def handle_power_boost_event(self, amount: int, source_card_id: str):
```

Comment:
I don't quite like the name handle_power_boost_event. You're not handling an event you're creating one.
You're also updating the artstatboosts.
Maybe just handle_power_boost is fine? I could also get behind just boost_power or power_boost.